### PR TITLE
add artifacthub-resource

### PR DESCRIPTION
**Pull request by PG2000**
Signed-off-by: Patrick Geschinski <PG2000@users.noreply.github.com>

### CODE REVIEW

**Comment by muntac** [comment]
@PG2000 Thanks!

File: pg2000-artifacthub-resource.yml
Comment (muntac) [inline pg2000-artifacthub-resource.yml]: @PG2000 small typo:
```suggestion
  Tracks and gets new versions of Helm Charts which are registered at https://artifacthub.io/
```

**Review by muntac — CHANGES_REQUESTED** [review]

**Comment by PG2000** [comment]
@muntac done.

**Review by muntac — APPROVED** [review]

**Comment by xtremerui** [comment]
@PG2000 this URL https://github.com/PG2000/artifacthub-resource seems broken as the repo is gone. Could we remove this resource type or you have some other plan? Thx.

**Comment by PG2000** [comment]
@xtremerui Hey, i moved the repo to another org and wait for approval until i can make it public again.
So i think it will be available again by the end of the week.
If you like i can open a new pr then.

What do you think?

**Comment by xtremerui** [comment]
@PG2000 good to know. We will remove it now to fix our CI and add it back later with updated url in your new PR.

**Pull request merged**
